src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -731,7 +731,7 @@ public UInt256 GetLatestPendingNonce(Address address)
             return maxPendingNonce;
         }
 
-        public IEnumerable<Transaction> GetBestTxOfEachSender() => _transactions.GetFirsts();
+        public Transaction GetBestTx() => _transactions.GetBest() ?? new();
```

Comment:
maybe nullable `Transaction? GetBestTx()`?

Reply:
+1 don't create artificial transaction, return null if not present